pkg/api/types/validation/form.go

```diff
@@ -461,3 +465,9 @@ func ValidateSelectOptions(options []*types.SelectOption, path *validation.Path)
 
 	return result
 }
+
+func ValidateFieldTypeBoolean(ftBoolean *types.FieldTypeBoolean, path *validation.Path) validation.ErrorList {
```

Comment:
Just a thought, but perhaps that a boolean field should not be a "key" field ?

Reply:
I also think it should be required as there's no clear UI difference between false and null. In the frontend I've made it always default to false, but this isn't enforced on the backend. Do you think I should add that constraint here too?

Reply:
I would think so.. I can't imagine a usecase where a boolean would be part of the key.. If we get some demands for it, we could "loosen" the constraint later on?